index.js

```diff
@@ -30,13 +30,25 @@ app.post('/signup', async (req, res) => {
     }
 })
 
+app.post('/login', async(req, res)=>{
+    try {
+        let previousUsers = await User.find({email: req.body.email, password: req.body.password, userName: req.body.userName });
+        let responseObj = { "Result" : "User not authorized." };
+        if (previousUsers.length == 1 ) {
+            responseObj.Result = "User is authorized!!";
+            res.send(responseObj);
+        }
+        else res.send(responseObj);
```

Comment:
No need for else block. Just save Result : "User Authorized"  inside if block and then after if res.send(responseObj);